src/Component/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,54 @@
+import {Link} from "react-router-dom";
+import React from  "react";
+function Navbar() {
+return ((<div>
+
+    
+    <nav class=" bg-green-7 text-color-0 pd-5 position-fixed t-0">
+        <div class="wt-80  m-auto  flex items-center">
+            <div class="text-xm">
+                <Link class="text-dec text-color-0 text-m cursor" to="/">Farmgistic</Link>
+            </div>
+            <div class="flex items-center justify-even m-lf">
+                <input type="search" placeholder="search"
+                    class="pd-y-3 pd-x-5 rounded-L border-none text-center outline-none" />
+                <i class="fa-solid fa-magnifying-glass bg-black-0 text-color-9 rounded-L pd-3 text-xm"></i>
+            </div>
```

Comment:
Replace attribute `class` with `className` in all occurrences in  this file

Reply:
Thanks for reminding me , actually forgot to replace. sure will add.